cluster_scaling_and_load_testing.md

```diff
@@ -0,0 +1,68 @@
+# Cluster Scaling and load testing
+
+## Cluster Scaling
+
+We will initially be starting out with 3 Control nodes and 5 worker nodes. From here will will scale up at intervals of 10 worker nodes per until we reach the maximum number of available nodes.
+
+## Interval Testing
+
+After each interval reports that the nodes are ready we will use [kube-burner][1]. The goal here is two-fold. First we want to make sure that we can schedule pods on each node, and second that we can max out node capacity without needing to add infra nodes to support the three controller nodes.
```

Comment:
> second that we can max out node capacity without needing to add infra nodes to support the three controller nodes.

Sorry, I don't understand what this means. Are we talking about adding nodes to the nerc-ocp-infra cluster?

Reply:
You can unload a lot of the admin tasks done by the controller nodes by adding infra nodes. These reduce the stress on the controllers, but can't be used to schedule regular workloads like worker nodes. I don't foresee us having any issues in the short term, but it's something that should be checked as the cluster and demand grows.

Reply:
ah, thanks for clarifying! It didn't occur to me that these are different class of nodes besides the worker and controllers.